docs/pages/management/operations/proxy-peering.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Proxy Peering Migration
+description: How to upgrade an existing Teleport cluster to Proxy Peering mode.
+---
+
+## Prerequisites
+
+An existing Teleport Enterprise cluster. See our [getting started guide](../../deploy-a-cluster/teleport-enterprise/getting-started.mdx) for help
+setting up a Teleport cluster.
+
+All components in the cluster should be running Teleport `10.0` or later. See our
+[upgrade procedure](./upgrading.mdx) for help upgrading a Teleport cluster.
+
+Teleport Proxy instances must be able to reach each other over the network on port
+3021 by default. Ensure there are no firewall policies that would block communication
+between instances.
+
+## Step 1/3. Enable Proxy Peering
+Update your cluster's Auth Service configuration to set the tunnel strategy type
+to `proxy_peering`.
+
+```yaml
+auth_service:
+    tunnel_strategy:
+        type: proxy_peering
+        agent_connection_count: 1
+```
+
+This setting will indicate to agents that they are only required to connect to 1
+Teleport Proxy instance as specified by the `agent_connection_count` field.
+
+## Step 2/3. Restart the Auth Services
+Restart all Teleport Auth Services running in the cluster to apply the new auth service
+configuration.
+
+## Step 3/3. Restart the Proxy Services
+Restart all Teleport Proxy instances running in the cluster in order for Proxy Peering
+specific services to start up.
```

Comment:
We should either say "Proxy Peering-specific" or rephrase so we won't need a compound modifier, e.g., "in order to start services related to Proxy Peering."